### ISSUE TITLE
[Bug][a11y]: Slider focus style should support forced-colors: active / WHCM

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3328]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Slider, SliderHandle

### Expected behavior

Slider focused style should match Spectrum specification and the focus ring should render with `forced-colors: active` or windows high contrast mode. We should also use `Highlight` color appropriately for track, fill-color and focused state.

#### Spectrum
![Spectrum Slider keyboard focused style](https://github.com/adobe/spectrum-web-components/assets/154077/1ea6e668-f6e2-486b-8a94-f7e1a2b08286)

#### Spectrum with forced-colors: active / WHCM
<img width="445" alt="Spectrum Slider keyboard focused style with forced-colors: active" src="https://github.com/adobe/spectrum-web-components/assets/154077/026e0da8-457d-4cba-8671-5af74898aa37">

### Actual behavior

Slider uses an older version of the focused style and the focus ring is not visible in high contrast mode.

#### SWC
<img width="445" alt="Spectrum Web Components Slider keyboard focused style" src="https://github.com/adobe/spectrum-web-components/assets/154077/83689344-bb7f-42e7-8c5b-d5ad6e7a07b9">

#### SWC with forced-colors: active / WHCM
<img width="445" alt="Spectrum Web Components Slider keyboard focused style with forced-colors: active" src="https://github.com/adobe/spectrum-web-components/assets/154077/756b36ed-da24-48a9-9c10-b27fd99d2c37">

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error

**Comment by Westbrook** [comment]
@pfulton this appears to occur in the Spectrum CSS source, as well.

**Comment by Westbrook** [comment]
@pfulton were yall able to capture this in Jira, or should I open an issue in GH to help yall along?

**Comment by pfulton** [comment]
Got it! PR is here and awaiting merge: https://github.com/adobe/spectrum-css/pull/2217

**Comment by Westbrook** [comment]
This was fixed along the way. 🎉
